### PR TITLE
chore: add e2e showing annotation for drafts

### PR DESCRIPTION
**Pull request by Nil20**
## Description

Clearly describe what has been changed. Include relevant context or background.
Explain how the issue was fixed (if applicable) and the root cause.

Link this pull request to the GitHub issue (and optionally name the branch `ocrvs-<issue #>`)

## Checklist

- [ ] I have linked the correct Github issue under "Development"
- [ ] I have tested the changes locally, and written appropriate tests
- [ ] I have tested beyond the happy path (e.g. edge cases, failure paths)
- [ ] I have updated the changelog with this change (if applicable)
- [ ] I have updated the GitHub issue status accordingly

### CODE REVIEW

File: e2e/testcases/application/draft-annotations.spec.ts [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    await selectDeclarationAction(page, 'Save & Exit', false)
+    await page.getByRole('button', { name: 'Confirm' }).click()
+  })
+
+  test('2.3 Re-open draft — annotation comment is visible in Annotations section', async () => {
+    await ensureOutboxIsEmpty(page)
+
+    await page.getByRole('button', { name: 'Drafts' }).click()
+
+    // const row = getRowByTitle(page, formattedName)
```
Comment (cibelius) [inline e2e/testcases/application/draft-annotations.spec.ts]:
```suggestion
```

**Review by cibelius — APPROVED** [review]
Nice 👍

**Comment by Nil20** [comment]
successful run: https://github.com/opencrvs/e2e/actions/runs/23000451670
